src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(8, 1)).toEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(40, 5)).toEqual([8, 8, 8, 8, 8]);
```

Comment:
Good work!) But pay attention to this case(should sort parts ascending if they are not equal)